### PR TITLE
[Feature] Connect-Accounts

### PR DESCRIPTION
**Pull request by Civolilah**

### CODE REVIEW

**Review by beganovich — CHANGES_REQUESTED** [review]
Looks good, minor changes needed.

File: src/pages/settings/bank-accounts/index/BankAccounts.tsx [hunk 1]
```diff
@@ -29,6 +30,21 @@ export function BankAccounts() {
...
 
+  const handleConnectAccounts = async () => {
+    const tokenResponse = await request(
+      'POST',
+      endpoint('/api/v1/one_time_token'),
+      { context: 'yodlee', platform: 'react' },
+      {
+        headers: {
+          'Content-Type': 'application/json',
+          'X-API-SECRET': 'password',
```
[the remark this answers is not in the document]
Comment (Civolilah) [reply src/pages/settings/bank-accounts/index/BankAccounts.tsx:41]: > 1. I am not sure we have X-Api-Secret?
> 2. We definitely don't wanna hardcode the secret, even if it exists.

Yeah, sure. I missed that. David sent me CURL and missed this part while copying. We don't need additional headers so I removed it.

File: src/pages/settings/bank-accounts/index/BankAccounts.tsx [hunk 2]
```diff
@@ -29,6 +30,21 @@ export function BankAccounts() {
...
+        },
+      }
+    );
+    window.open(`/yodlee/onboard/${tokenResponse?.data?.hash}`);
```
[the remark this answers is not in the document]
Comment (Civolilah) [reply src/pages/settings/bank-accounts/index/BankAccounts.tsx:45]: > For dealing with param replacements, make sure to use route helper.

changed

**Review by beganovich — APPROVED** [review]
Looks good, thanks :+1:

**Pull request merged**